src/components/InfoAsignacionDialog/index.js

```diff
@@ -28,37 +29,82 @@ InfoAsignacionDialog.propTypes = {
   onChange: propTypes.func.isRequired,
   aplicarCambios: propTypes.func.isRequired,
   desasignar: propTypes.func.isRequired,
+  project: propTypes.object.isRequired,
 };
 
 function InfoAsignacionDialog({
   asignacionInfo,
   projectName,
   personName,
+  project,
   onClose,
   onChange,
   aplicarCambios,
   desasignar,
 }) {
   const classes = useStyles();
 
-  const cargasHorariasItems = cargasHorarias_t.map((cargaHoraria, id) => {
+  const renderColor = (state) => {
+    let color = "";
+    switch (state) {
+      case "verde":
+        color = "#7EDE6D";
+        break;
+      case "rojo":
+        color = "#E87272";
+        break;
+      case "amarillo":
+        color = "#FAE269";
+        break;
+      case "upcoming":
+        color = "#B0CFCB";
+        break;
+    }
```

Comment:
This is duplicated in another component, right?
I would create a helper function isolated to manage this